lib/service/commands/list.rb

```diff
@@ -9,13 +9,34 @@ module List
 
       TRIGGERS = [nil, "list", "ls"].freeze
 
-      def run
+      def run(json: nil)
         formulae = Formulae.services_list
         if formulae.blank?
           opoo "No services available to control with `#{Service::ServicesCli.bin}`" if $stderr.tty?
           return
         end
-        print_table(formulae)
+
+        if json
+          print_json(formulae)
+        else
+          print_table(formulae)
+        end
+      end
+
+      JSON_FIELDS = [:name, :status, :user, :file, :exit_code].freeze
```

Comment:
What are we using this for?

Reply:
This array is used to limit the JSON output to the same fields that are displayed when the user uses the `brew services list` command. There already exists the `brew services info --all --json` command which displays all information about all services so if we don't actually care about having more verbose JSON output we could just redirect `brew services list --json` to that command which would be a lot easier imo. The problem is that that command includes three times the number of fields.